docs/Documentation/Applications/comsol.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Comsol
+---
+
+# Using COMSOL Software on the Eagle System
+COMSOL Multiphysics is a versatile finite element analysis and simulation package. The COMSOL Desktop GUI (graphical user interface) offers an environment for building and solving models. It provides script-based modeling capabilities.
+
+Currently, we host three floating network licenses and a number of additional modules. Issue the command lmstat.comsol to see current license status and COMSOL modules available.
+
+Building a COMSOL Model
+Extensive documentation is available in the menu: Help > Documentation and in Help > Dynamic Help. For beginners, it is highly recommended to follow the steps in Introduction to COMSOL Multiphysics found in Help > Documentation.
```

Comment:
I'm not sure what menu this is referring to. If its on the CMS, we should change the instructions to reflect where it will be on Github. Or, link to the website that this is on.